Tasks/VsTest/task.loc.json

```diff
@@ -165,7 +165,14 @@
     "PowerShell": {
       "target": "$(currentDirectory)\\VSTest.ps1",
       "argumentFormat": "",
-      "workingDirectory": "$(currentDirectory)"
+      "workingDirectory": "$(currentDirectory)",
+      "platforms": [
+        "windows"
+      ]
+    },
+    "Node": {
```

Comment:
great

---
In reply to: [65325909](https://github.com/Microsoft/vsts-tasks/pull/1817#discussion_r65325909) [](ancestors = 65325909)